stock_packaging_usability_pp/readme/USAGE.rst

```diff
@@ -0,0 +1,5 @@
+This module modifies the behavior of the buttons *Put in new
+pack* on operations (button provided by the module
+*stock_packing_usability*) and *Put in pack* on pickings (native button):
+when you click on those buttons, Odoo will ask you to select the
+packaging (object *product.packaging*) of the package.
```

Comment:
This doesn't seem like an appropriate usage section. AFAIK it should contain a step by step guide on how to exercise this module's features. Check out [the instructions in the template](https://github.com/OCA/maintainer-tools/blob/master/template/module/readme/USAGE.rst).

This belongs more to a description or context. But, if I read these instructions, I don't know where to go or what to do to test the module on runboat.

Reply:
I agree with your remark and I made the change